src/Components/SuperAdmins/index.js

```diff
@@ -1,10 +1,34 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import List from './List/List';
 import styles from './super-admins.module.css';
 
 function SuperAdmins() {
+  const [admins, adminsList] = useState([]);
+
+  useEffect(() => {
+    fetch(`${process.env.REACT_APP_API_URL}/admins`)
+      .then((response) => response.json())
+      .then((response) => {
+        adminsList(response.data);
+      });
+  }, []);
+
+  const handleDelete = async (id) => {
+    await fetch(`${process.env.REACT_APP_API_URL}/admins/${id}`, {
+      method: 'DELETE'
+    });
+    adminsList([...admins.filter((admins) => admins._id !== id)]);
+  };
+
   return (
-    <section className={styles.container}>
-      <h2>SuperAdmins</h2>
-    </section>
+    <div className={styles.container}>
+      <h3>Admins List</h3>
+      <a href={`super-admins/form`}>
```

Comment:
```suggestion
      <a href={'super-admins/form'}>
```
Replace backsticks (`) with (') as you are not using a template literal syntax

Reply:
done